portality/templates/doaj/includes/_journal_meta_description.html

```diff
@@ -0,0 +1,5 @@
+{%- if bibjson.keywords -%}
```

Comment:
I can't see `bibjson` in the scope for this.  It would probably be better to do a `journal.bibjson()` at the top of this like is done for the article snippets.

Same for the other file _journal_meta_title.html

Reply:
Done in https://github.com/DOAJ/doaj/pull/2220/commits/9d6752ffc89fe7f8bb9279a8b4ff788d7e81060e! Thank you